packages/web/config/ibc-assets.ts

```diff
@@ -784,6 +784,13 @@ export const IBCAssetInfos: (IBCAsset & {
     //   "https://blue.kujira.app/ibc?destination=kaiyo-1&source=osmosis-1&denom=ukuji",
     isVerified: true,
   },
+  {
+    counterpartyChainId: "acre_9052-1",
+    sourceChannelId: "channel-490",
+    destChannelId: "channel-0",
+    coinMinimalDenom: "aacre",
+    isVerified: true,
```

Comment:
please remove the 'isVerified' property (this is used to display on Osmosis Main, but this asset has not yet been approved.

Reply:
Done - https://github.com/osmosis-labs/osmosis-frontend/pull/1085/commits/3fe56229141767847e9cf12660ae72714dea7d1f